pages/blog/release-notes-3.0.0.md

```diff
@@ -0,0 +1,329 @@
+```
+Take notice of how `description` is not overwritten by the traits:
+
+```
+messageId: userSignup
+name: UserSignup
+title: User signup
+summary: Action to sign a user up.
+description: A longer description. # it's still description from "main" object
+payload:
+  $ref: '#/components/schemas/userSignupPayload'
+```
+
+Issues: [#505](https://github.com/asyncapi/spec/issues/505) | Pull request: [#517](https://github.com/asyncapi/spec/pull/517), [#532](https://github.com/asyncapi/spec/pull/532), [#907](https://github.com/asyncapi/spec/pull/907)
+
+### Schema format and payload definition
+With schemas, one thing that has always been impossible was reusing schemas with different schema formats. That's because the schema format information is part of the message object. That means that if you reference a Schema object, it has no information about the schema format because it's not located together.
+
+In v3, schemaFormat has been removed from the message object and message trait object, and a new schema Object called `Multi Format Schema Object` has been introduced which pairs a schema together with its schema format. Which now enables much better reusability:
+
+```
+asyncapi: 3.0.0
+components:
+  schemas:
+    avroSchema:
+      schemaFormat: 'application/vnd.apache.avro+yaml;version=1.9.0'
+      schema:           
+        type: record
+        name: User
+        namespace: com.company
+        doc: User information
+        fields:
+          - name: displayName
+            type: string
+```
+
+Issues: [#622](https://github.com/asyncapi/spec/issues/622) | Pull request: [#797](https://github.com/asyncapi/spec/pull/797), [#910](https://github.com/asyncapi/spec/pull/910)
+
+### Simplified Parameters
+In v2, it was possible to use the full power of JSON Schema to define parameters, however, it introduced a lot of complexity to parameters, so for v3 it was dialed way down to only allow a very small set of properties.
+Parameters can now only have the following properties: `enum`, `default`, `description`, `examples`, and `location`. 
+
+```
+asyncapi: 3.0.0
+channels: 
+  userSignup:
+    address: user/{userId}/signedup
+    parameters:
+      userId:
+        description: Id of the user.
+```
+
+Issues: [#583](https://github.com/asyncapi/spec/issues/583) | Pull request: [#935](https://github.com/asyncapi/spec/pull/935) | Specification information: https://www.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#parameterObject
+
+### Editorial changes
+
+We have [removed the note that stated we strived to be compatible with OpenAPI where possible]([#933](https://github.com/asyncapi/spec/pull/933)), because, with the recent changes, this is no longer the case. That said we of course still strive to make the different specs as interoperable as possible i.e. with Avro, RAML, OpenAPI Schema, etc. 
+
+## Tooling support
+The following official AsyncAPI tools are already updated to support v3:
+
+TODO
```

Comment:
What is left TODO here? Please fix or delete this as needed 😄 @jonaslagoni

Reply:
@alequetzalli as also stated in the PR, this is just the first draft, the last and final details will be done the closer to release we are 🙂 

Dont worry, this wont be present for the final release 😄 

But we can remove the `TODO` if you really don't want it present 😄

Reply:
Ah, since that was a while ago, I thought that you pinging me for a review here last week meant this was 💯 done. I didn’t realize it was still a work in progress and thought it was accidentally left.

In that case, def no worries on having that placeholder reminder.